### ISSUE TITLE
I have a question about the deep_sort_app.py file. Is this file can run multiple MOT sequence? 

### ISSUE THREAD

**Issue by 404hasbeenfound** [nwojke/deep_sort#146]
i raise this question because your readme run

python deep_sort_app.py \
    --sequence_dir=./MOT16/test/MOT16-06 \
    --detection_file=./resources/detections/MOT16_POI_test/MOT16-06.npy \
    --min_confidence=0.3 \
    --nn_budget=100 \
    --display=True

the sequence_dir and the detection_file are single.

**Comment by nwojke** [comment]
You are right, the script ``deep_sort_app.py`` only runs on a single detection. If you want to evaluate multiple sequences, you will have to write a wrapper similar to [this file](https://github.com/nwojke/deep_sort/blob/master/evaluate_motchallenge.py).
